client/packages/electron/src/electron.ts

```diff
@@ -34,8 +35,39 @@ class DiscoveryRunner {
   }
 }
 
+class BarcodeScanner {
+  device: HID.HID | undefined;
+
+  constructor() {
+    this.findDevice();
+  }
+
+  findDevice() {
+    try {
+      this.device = new HID.HID(1504, 2194);
```

Comment:
Add comment about vendor and product id? e.g. what they translate to?

Reply:
thanks - yes, I had meant to comment on this.
I've just popped in the ids for the zebra scanner that we ( currently me ) have. will update.